### PR TITLE
Fix funded bp full profile link

### PR DESCRIPTION
**Pull request by mcstover**
- Removes our older FundedBorrower Profile presentation
- Sets the new Funded BP as the default
- Removes exp code
- Fixes the full profile link to navigate to `/lend-classic`
- Fixes some undefined data fields and conditional values

### CODE REVIEW

File: src/pages/BorrowerProfile/BorrowerProfile.vue [hunk 1]
```diff
@@ -491,7 +494,7 @@ export default {
 			this.countryName = loan?.geocode?.country?.name ?? '';
 			this.hash = loan?.image?.hash ?? '';
 			this.numLenders = loan?.lenders?.totalCount ?? 0;
-			this.endDate = format(parseISO(loan?.plannedExpirationDate), 'M/d') ?? '';
+			this.endDate = loan?.plannedExpirationDate ? format(parseISO(loan?.plannedExpirationDate), 'M/d') : '';
```
Comment (mcstover) [inline src/pages/BorrowerProfile/BorrowerProfile.vue:497]: This was causing an issue b/c `plannedExpirationDate` is null on a funded loan.

File: src/pages/BorrowerProfile/BorrowerProfile.vue [hunk 2]
```diff
@@ -409,6 +411,7 @@ export default {
 			inviterName: '',
 			inviterIsGuestOrAnonymous: false,
 			inPfp: false,
+			itemsInBasket: [],
```
Comment (mcstover) [inline src/pages/BorrowerProfile/BorrowerProfile.vue:414]: This along with the 2 above were causing issues b/c they weren't defined on data and consequently weren't reactive.

File: src/components/BorrowerProfile/LoanProgress.vue [hunk 3]
```diff
@@ -23,7 +23,7 @@
 					</p>
 					<p class="tw-text-h4 tw-text-action tw-block">
 						<router-link
-							:to="`/lend/${$route.params.id}?minimal=false`"
+							:to="`/lend-classic/${$route.params.id}?minimal=false`"
```
Comment (mcstover) [inline src/components/BorrowerProfile/LoanProgress.vue:26]: new destination link for full funded bp

**Review by dyersituations — APPROVED** [review]

**Review by emuvente — APPROVED** [review]
Thanks for the cleanup and the fixes!

**Pull request merged**